back/config/environments/development.rb

```diff
@@ -19,11 +21,12 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Enable server timing
+  config.server_timing = true
+
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
   if Rails.root.join('tmp/caching-dev.txt').exist?
-    config.action_controller.perform_caching = true
```

Comment:
I'm not quite sure what this is doing, the docs are unspecific: 

> Configures whether the application should perform the caching features provided by the Action Controller component or not. Set to false in the development environment, true in production. If it's not specified, the default will be true.

https://guides.rubyonrails.org/configuring.html#config-action-controller-perform-caching